apps/analysis/tests/test_mutations.py

```diff
@@ -1326,3 +1326,204 @@ def _query_check(_id, **kwargs):
             else:
                 self.force_login(user)
             assert _query_public_snapshot_check(snapshot_slug)['data']['publicAnalysisReportSnapshot'] is not None
+
+
+class TestAnalysisMutationSchema(GraphQLTestCase):
+    CREATE_MUTATION = '''
+        mutation MyMutation($analysisData: AnalysisInputType!, $projectId: ID!) {
+          project(id: $projectId) {
+            analysisCreate(
+              data: $analysisData
+            ) {
+              errors
+              ok
+              result {
+                id
+                endDate
+                title
+                teamLead {
+                  id
+                }
+              }
+            }
+          }
+        }
+    '''
+    UPDATE_MUTATION = '''
```

Comment:
Let's add newline between large code blocks
```suggestion

    UPDATE_MUTATION = '''
```